swagger/v1/swagger.yaml

```diff
@@ -255,9 +278,9 @@ paths:
                 error_class: ActiveRecord::RecordNotFound
                 message: 'No such proceeding type: ''XX999'''
                 backtrace:
-                - "/Users/stephenrichards/.rbenv/versions/3.1.1/lib/ruby/gems/3.1.0/gems/activerecord-7.0.2.4/lib/active_record/relation/finder_methods.rb:378:in
+                - "/Users/stephenrichards/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/activerecord-7.0.2.4/lib/active_record/relation/finder_methods.rb:378:in
```

Comment:
I assume this was an error that has been recorded and needs removing?

Reply:
Yeah, I'm not sure an API needs to return a backtrace, the `No such proceeding type: ''XX999''` seems fine to me